python/mvdtool.cpp

```diff
@@ -143,20 +143,34 @@ PYBIND11_MODULE(mvdtool, mvd) {
                 f.openComboTsv(filename);
              },
              "filename"_a)
+        .def("positions", [](const File& f, int offset) {
```

Comment:
Isnt there a conflict with the overload below? The signature bellow also allows a single param.

Reply:
As discussed offline, pybind sets the precedence in this case based on the order that the functions where declared. I decided to specifically declare the functions for all the cases to avoid any conflict in the future